docs/AutomationReference.md

```diff
@@ -123,26 +123,46 @@ The `ScannerFactory.CreateScanner` method returns an `IScanner` object.
 
 #### IScanner
 
+##### `ScanAsync`
+The `ScanAsync` method asynchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner. This method should be `await`ed.
+
+###### Parameters
+`IScanner.ScanAsync` accepts two parameters: an instance of `ScanOptions` containing custom settings for this scan (or `null` for default options), and a [`CancellationToken`](https://learn.microsoft.com/en-gb/dotnet/api/system.threading.cancellationtoken?view=netstandard-2.0).
+
+###### Return object
+`ScanAsync` returns a `ScanOutput` object.
+
 ##### `Scan`
-The Scan runs AxeWindows automated tests using the config provided at the time of creation of the scanner.
+The `Scan` method synchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner, and blocks until the scan is complete.
 
 ###### Parameters
+`IScanner.Scan` accepts one parameter: an instance of `ScanOptions` containing custom settings for this scan (or `null` for default options).
```

Comment:
Where possible, let's use internal links. Instead of `ScanOptions`, it could be [`ScanOptions`](https://github.com/microsoft/axe-windows/blob/main/docs/AutomationReference.md#scanoptions)